api/test/integration/env/.env

```diff
@@ -0,0 +1 @@
+ENV_MODE=cluster
```

Comment:
We should remove this file

Reply:
Keep in mind that the Jenkins pipeline does not include the `ENV_MODE=cluster && docker-compose build ...` in the command launched from Ansible playbook. It is possible to change the ansible file to avoid creating the .env file.

Reply:
But Jenkins is currently working fine without including the `ENV_MODE=cluster`, isn't it? I don't think we need to add this, the env_mode is designed to work with the [run_tests.py](https://github.com/wazuh/wazuh/blob/master/api/test/integration/run_tests.py) script.